CHANGELOG.latest.md

```diff
@@ -1 +1,34 @@
-- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum.
\ No newline at end of file
+**Note:** This release is based off of 5.0.0-rc3. These are the changes compared to 5.0.0-rc3:
+
+- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum
+    https://github.com/RevenueCat/purchases-android/pull/506
+- Added retries with exponential backoff when reconnecting to Google's billing service
+    https://github.com/RevenueCat/purchases-android/pull/489
+- Updated to BillingClient 4.1
+    https://github.com/RevenueCat/purchases-android/pull/500
+- Prevent Amazon from swallowing exceptions
+    https://github.com/RevenueCat/purchases-android/pull/499
+- Improvements on Amazon implementation preventing a lot of issues related to price and currency parsing
+    https://github.com/RevenueCat/purchases-android/pull/503
+
+[Full Changelog](https://github.com/revenuecat/purchases-android/compare/main...5.0.0)
+
+### Migration Guide
+- A number of renames, removal of some deprecated functions, and a change to configuration. Amazon Store support is also included Checkout the full list of changes in [migrations/v5-MIGRATION.md]
+
+### API changes:
+- See our [RevenueCat V5 API update doc]([migrations/v5-MIGRATION.md]) for API updates.
+
+#### Amazon Support
```

Comment:
kinda feel like this deserves to be `###`, since it's the highlight of the release

Reply:
should we be more specific and call it Amazon Store support?
Also, I think we could link to Amazon Store itself, as well as our docs for it